### PR TITLE
Render completed transactions from content store

### PR DESCRIPTION
**Pull request by brenetic**
This format has now been migrated.

https://trello.com/c/unTNbCHi/658-migrate-completed-transaction-format-to-rendered

Mobbed with @emmabeynon @binaryberry

### CODE REVIEW

File: test/integration/completed_transaction_test.rb [hunk 1]
```diff
@@ -2,11 +2,32 @@
 require 'integration_test_helper'
 
 class CompletedTransactionTest < ActionDispatch::IntegrationTest
+  setup do
+    @payload = {
```
Comment (davidbasalla) [inline test/integration/completed_transaction_test.rb]: We could remove some of the details here to make it easier to read, as I'm guessing we don't test for each one (eg analytics_identifier). Not sure what we've done previously for the other formats

**Comment by davidbasalla** [comment]
We could clean up the PublicationPresenter a little bit now that this is done, as it has a number of promotion related methods. Looks great though! 👍

File: app/views/completed_transaction/show.html.erb [hunk 2]
```diff
@@ -8,34 +8,36 @@
...
-              data: {
-                module: 'track-click',
-                track_category: 'linkTrack',
-                track_action: 'linkClicked',
-                track_label: @publication.promotion_url
-              } %>
-      </p>
-    </div>
+  <% if @publication.promotion %>
+    <% if @publication.promotion['category'] == 'organ_donor' %>
```
Comment (davidbasalla) [inline app/views/completed_transaction/show.html.erb]: What was your reasoning for not using the presenter methods here? I don't have any strong opinions in it, just curious

File: test/integration/completed_transaction_test.rb (hunk 1 above)
Comment (emmabeynon) [inline test/integration/completed_transaction_test.rb]: Thanks - we've removed the unnecessary stuff, and kept what's required ⚡️

File: app/views/completed_transaction/show.html.erb (hunk 2 above)
Comment (emmabeynon) [inline app/views/completed_transaction/show.html.erb]: There aren't any methods available in the CompletedTransactionPresenter.  @binaryberry and I did discuss extracting this out in to methods, but felt there wasn't enough duplication going on to warrant it.

File: test/integration/completed_transaction_test.rb (hunk 1 above)
Comment (davidbasalla) [inline test/integration/completed_transaction_test.rb]: Great, could do a similar clean up in 'test/functional/completed_transaction_controller_test.rb' to be consistent?

Comment (emmabeynon) [inline test/integration/completed_transaction_test.rb]: Should now be taken care of 👍

**Pull request merged**